From 58d6298a94339370f9de61768b25b8c9a7802028 Mon Sep 17 00:00:00 2001
From: Gerd Hoffmann <kraxel@redhat.com>
Date: Wed, 5 Jan 2011 15:29:33 -0200
Subject: [PATCH 28/48] spice-core: fix watching for write events

RH-Author: Gerd Hoffmann <kraxel@redhat.com>
Message-id: <1294241382-17988-30-git-send-email-kraxel@redhat.com>
Patchwork-id: 15761
O-Subject: [RHEL-6 kvm PATCH 29/38] spice-core: fix watching for write events
Bugzilla: 642131 634153 615947 632458 631832 647865
RH-Acked-by: Uri Lublin <uril@redhat.com>
RH-Acked-by: Alon Levy <alevy@redhat.com>
RH-Acked-by: Hans de Goede <hdegoede@redhat.com>

From: Hans de Goede <hdegoede@redhat.com>

upstream: 3d6d306c104abe37610184f12e9342fcbc078395

Signed-off-by: Gerd Hoffmann <kraxel@redhat.com>
---
 ui/spice-core.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

Signed-off-by: Luiz Capitulino <lcapitulino@redhat.com>
---
 ui/spice-core.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/ui/spice-core.c b/ui/spice-core.c
index 9ea507c..ab4a0cb 100644
--- a/ui/spice-core.c
+++ b/ui/spice-core.c
@@ -95,7 +95,7 @@ static void watch_update_mask(SpiceWatch *watch, int event_mask)
         on_read = watch_read;
     }
     if (watch->event_mask & SPICE_WATCH_EVENT_WRITE) {
-        on_read = watch_write;
+        on_write = watch_write;
     }
     qemu_set_fd_handler(watch->fd, on_read, on_write, watch);
 }
-- 
1.7.4.rc1.16.gd2f15e

